internal/plugin/connectors/tcp/proxy_service.go

```diff
@@ -119,7 +119,7 @@ func (proxy *proxyService) handleConnection(clientConn net.Conn) error {
 	// immediately zeroize credentials after connecting
 	internal.ZeroizeCredentials(backendCredentials)
 
-	logger.Debugf("Connection opened on %v to %v.\n", clientConn.LocalAddr(), targetConn.RemoteAddr())
+	logger.Debugf("Proxying connection on %v to %v.\n", clientConn.LocalAddr(), targetConn.RemoteAddr())
```

Comment:
This is imo pretty important debugging info when things break. This only shows the IP and is only in debug mode too.

> comes from the credentials

That delineation is only because we have it in that section in yml with all the other variables, not because it's a credential per-se.

Reply:
we've gotten feedback that we should treat it securely; imo this line should be removed or at least only log the localaddr (which should be known, right?)

Reply:
> we've gotten feedback that we should treat it securely

I feel like this might be one of those things that the benefits are too high to remove this for the operator persona (shows that the DNS resolution is working correctly). Information leakage from the server IP is realistically unusable without  authentication details _and_ you have to turn debugging on manually so I don't quite know what we are protecting by removal of this. Also was this feedback internal or external to corp? 

> at least only log the localaddr

I don't think there's a point in just logging the localaddr imo but I'd rather we have something than nothing.

Reply:
it was internal feedback, and I expect with this line we'll be asked to remove it.